app/models/user.rb

```diff
@@ -251,4 +253,8 @@ def make_salt
     def secure_hash(string)
       Digest::SHA2.hexdigest(string)
     end
+
+    def default_country_to_usa
+      self.country ||= "USA"
```

Comment:
you're doing this in the DB already, why do it in the app too?

Reply:
I set all legacy users to have the country `USA` in the database, but didn't set a default. This handles new/edited users until such a time as it's specified on creation/editing.